site/styles/_forms.scss

```diff
@@ -112,6 +112,19 @@ s72-activatedevice-form,
       color: rgba(var(--body-color-rgb), 0.8);
     }
   }
+
+  &.s72-form-control-readonly,
```

Comment:
These mean you can simply set an input as readonly and it will look like a readonly text field. You won't need to add a whole new css rule.